models/rubicon/base/rubicon_base_offers.sql

```diff
@@ -0,0 +1,227 @@
+{{ config(
+    schema = 'rubicon_base',
+    tags = ['dunesql'],
+    alias = alias('offers'),
+    partition_by = ['block_date'],
```

Comment:
can we either:
- remove partitions universally
- or add new column `block_month` and apply as partition universally

Reply:
hey @jeff-dude i have gone through and updated this to drive off of `block_month` since this is what it looks like `dex.trades` will soon be doing. if you have time, could you explain what is going on within `partition_by` and why `block_month` is better than `block_date`?

Reply:
the partitions will split parquet files in the data lake by what you assign in the config. the larger the partition, the better (there's some trial/error here, our short term approach is to settle at monthly level). then when dunesql queries the data lake, if you filter/group by on the partition column, it will help read performance by telling the engine which files to read in the lake, skipping over all the other partitions